treenode.go

```diff
@@ -679,6 +681,9 @@ func (n *TreeNodeInstance) SetConfig(c *GenericConfig) {
 	n.configMut.Lock()
 	defer n.configMut.Unlock()
 	n.config = c
+	for id := range n.sentTo {
+		n.sentTo[id] = false
```

Comment:
The default value for a boolean is already "false" so I think this initialization step can be avoided.